app/views/homepage/index.html.erb

```diff
@@ -128,30 +128,30 @@
         </div>
 
         <div class="govuk-grid-column-one-third">
-          <a href="/government/topical-events/budget-2020" aria-hidden="true" tabindex="-1">
-            <%= image_tag "homepage/budget-2020--480w.jpg", {
+          <a href="/government/topical-events/a-plan-for-jobs-2020" aria-hidden="true" tabindex="-1">
+            <%= image_tag "homepage/plan-for-jobs-2020--480w.png", {
               alt: "",
```

Comment:
Shouldn't get give it an alt name for accesibility?
```suggestion
              alt: "A Plan for Jobs",
```

Reply:
I think it's arguable that as the "image" is text repeating the linked text, adding this alt-text would mean the user gets repeated text read out.

Reply:
Agreed - I think that the blank alt text is acceptable here as the suggestion is repeating the heading further down the page. Anyone using a screen reader or similar assistive tech wouldn't gain anything by having this alt text read out.